docs/general/builtins/scalar-functions.rst

```diff
@@ -2037,7 +2037,7 @@ Returns: Same as input type.
 
 ::
 
-    cr> select exp(1.0) AS exp;
+    cr> select exp(1.0) AS exp; # doctest: +SKIP
```

Comment:
don't know if it reads better to change the prefix to something like: `crash>` or `cratedb>`, but that may be misleading since, it's not the prompt users see when running crash.